hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1568,14 +1568,14 @@
             - Write the KEY_IV_DATA_IN_CLEAR and DATA_OUT_CLEAR trigger bits to 1 and wait for it to
               complete by polling the status idle bit.
             - Read back the data out CSRs - they should all read garbage values.
-            - Assertion check verifies that the internal states (data_in, key share and IV are also
-              garbage, i.e. different from the originally written values.
+            - Assertion check verifies that the IV are also garbage, i.e. different from the originally
+              written values.
             - Assertion checks proves that all interfaces are connected across AST RNG, ES, CSRNG,
```

Comment:
I think this can not be done by software. @vogelpi am I right?

Reply:
Thanks Pirmin, @rswarbrick could you please let me know your opinion on that?
In case it should be done by dv, should I remove this step from the test plan?